### PR TITLE
Update cypress to 13.2.0

### PR DESCRIPTION
**Pull request by wimdetroyer**
Update testcontainers to 1.19.0
Add mac troubleshooting section

### CODE REVIEW

**Comment by wimdetroyer** [comment]
I have tested this update on our end, and everything seems to work. Great job on this library!

Due to https://github.com/testcontainers/testcontainers-java/pull/7421  I also had to make some changes in the tests.

If you think this PR is OK to be merged into the master branch, would you be so kind to label this PR with 'hacktoberfest-accepted' please? https://hacktoberfest.com/ is currently going on.

File: README.adoc [hunk 1]
```diff
@@ -334,6 +333,10 @@ The `CYPRESS_RECORD_KEY` environment variable needs to be set for this to work.
 |===
 |Testcontainers-cypress |Testcontainers | Cypress
 
+|https://github.com/wimdeblauwe/testcontainers-cypress/releases/tag/testcontainers-cypress-1.9.0[1.9.0]
+|https://github.com/testcontainers/testcontainers-java/releases/tag/1.19.0[1.19.0]
```
Comment (wimdeblauwe) [inline README.adoc:337]: I see 1.19.1 is released yesterday. Maybe update here and in the `pom.xml` to that version?

File: README.adoc [hunk 2]
```diff
@@ -334,6 +333,10 @@ The `CYPRESS_RECORD_KEY` environment variable needs to be set for this to work.
...
+|https://docs.cypress.io/guides/references/changelog.html#13-2-0[13.2.0]
```
Comment (wimdeblauwe) [inline README.adoc:338]: There is now a 13.3.0 I noticed. Maybe update to that one https://docs.cypress.io/guides/references/changelog#13-3-0 ?

**Comment by wimdeblauwe** [comment]
Thank you for this PR! If you can update the versions to the latest ones available, we can merge this and I can do a new release.

File: README.adoc (hunk 1 above)
Comment (wimdetroyer) [reply README.adoc]: Done, thanks

File: README.adoc (hunk 2 above)
Comment (wimdetroyer) [reply README.adoc]: Ofcourse!

**Comment by wimdetroyer** [comment]
I've bumped the versions again and did a test again on our project. Everything still seems to work 😃 

I didn't find a way to squash the commits in this PR though. Apparently you can't do that if you fork a repo ? 

Will you be able to squash to a clean commit? If not I can just reopen a new PR

**Pull request merged**